### PR TITLE
Add Groestlcoin

### PR DESCRIPTION
**Pull request by gruve-p**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add Groestlcoin back in

### CODE REVIEW

**Comment by gruve-p** [comment]
@bmoore-msft This PR is ready for you to review.

File: application-workloads/blockchain/blockchain/metadata.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (bmoore-msft) [inline application-workloads/blockchain/blockchain/metadata.json:1]: update this file to reflect that only groestlcoin is supported

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/blockchain/blockchain/metadata.json (hunk 1 above)
Comment (gruve-p) [reply application-workloads/blockchain/blockchain/metadata.json:1]: where do you want this info? In the summary or description? Maybe you can add a suggestion

Comment (gruve-p) [reply application-workloads/blockchain/blockchain/metadata.json:1]: @bmoore-msft ^

Comment (gruve-p) [reply application-workloads/blockchain/blockchain/metadata.json:1]: Done

**Pull request merged**